static/src/javascripts/projects/commercial/modules/third-party-tags/imr-worldwide.spec.js

```diff
@@ -31,12 +31,25 @@ jest.mock('lib/config', () => {
     return Object.assign({}, defaultConfig, {
         get: (path: string = '', defaultValue: any) =>
             path
-                .replace(/\[(.+?)\]/g, '.$1')
```

Comment:
wow how was this working before?

Reply:
should the closing `]` really not be escaped?

Reply:
it has been marked as useless escaped character by my IDE, so I've tried to remove it and seem to work fine. Made this change only in tests, although exact same code exists in implementation code as well

Reply:
works for me!